docs/userguide/products/fault-association.html

```diff
@@ -0,0 +1,242 @@
+probabilities apply.
+The feature collection comprises of one or more fault features and their association probabilites and distance from the event.
+
+<h3>Metadata Object</h3>
+<dl>   
+                <dt>process_time</dt>
+                <dd>The date time the fault association probabilties were created</dd>
+
+                <dt>software_version</dt>
+                <dd>The version of Eq2CFM that was used to compute the probabilites</dd>
+
+                <dt>cfm_version</dt>
+                <dd>The version of the SCEC Community Fault Model that was used to compute the probabilites</dd>
+                </dd>
+              
+        </dl>
+
+<h3>Result Object</h3>
+<dl>    
+                <dt>probability</dt>
+                <dd>The probability in percent the earthquake is associated with this fault</dd>
+        
+                <dt>title</dt>
+                <dd>This could be the name of an individual fault or a group of faults</dd>
+        
+                </dd>
+        </dl>
+<h3>Event Feature</h3>
+<dl>
+                <dt>id</dt>
+                <dd>Event id - this is an id in ComCat</dd>
+
+</dl>
+<h4>Event Feature properties</h4>
+<dl>
+                <dt>class</dt>
+                <dd>Identifies what feature is being described. In this product there are 2 classes "Event" and "Fault"</dd>
+
+                <dt>mag</dt>
+                <dd>The magnitude of the earthquake</dd>
+
+                <dt>title</dt>
+                <dd>A string describing the earthquake - usually gives some geographical reference to users.</dd>
+
+                <dt>utctime</dt>
+                <dd>date string of origin time in UTC</dd>
+
+                <dt>version</dt>
+                <dd>event version given by the network operator</dd>
+
+                </dd>
+        </dl>
+
+<h3>Fault Feature</h3>
+<dl>    
+                <dt>id</dt>
+                <dd>SCEC CFM fault id</dd>
+  
+</dl>
+<h4>Fault Feature properties</h4>
+<dl>
+		<dt>distance</dt>
+		<dd>distance from epicenter in kilometers</dd>
+
+		<dt>probability</dt>
+		<dd>The probability in percent the earthquake is associated with this fault</dd>
+
+		<dt>name</dt>
+		<dd>SCEC CFM 5.0 Fault name and closest segment if available.</dd>
+
+		<dt>qfaultids</dt>
+		<dd>an array of corresponding faults ids in the QFault database.</dd>
+
+		</dd>
+	</dl>
+
+<h3>Example</h3>
+<div class="code"><pre> {
+  "metadata": {
+    "process_time": "2021-04-13T21:40:03Z",
+    "software_version": "NA",
+    "cfm_version": "5"
+    },
+  "result": [
+    {
+      "probability": 37,
+      "title": "Not in CFM"
+    },
+    {
+      "probability": 35,
+      "title": "Berdoo Canyon fault"
+    },
+    {
+      "probability": 28,
+      "title": "Other CFM faults"
+    }
+  ],
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "event",
```

Comment:
I thought we discussed moving the `event` feature to `metadata`, so the feature collection would only be faults.